mat-process-api.Tests/V1/Validators/GetProcessRequestValidatorTest.cs

```diff
@@ -19,28 +19,45 @@ public void SetUp()
             validator = new GetProcessDocumentRequestValidator();
         }
 
-        [Test]
         [TestCase(null)]
         [TestCase("")]
         [TestCase(" ")]
-        public void given_null_or_empty_or_whitespace_processRef_the_validator_should_return_an_error(string processRef)
+        public void given_null_or_empty_or_whitespace_processRef_the_GetProcessDocumentRequestValidator_should_return_an_error(string processRef)
```

Comment:
You don't need this test anymore, because you have it full covered with 2 tests bellow:
`given_null_processRef_the_GetProcessDocumentRequestValidator_should_return_a_correct_error_message` and
`given_empty_or_whitespace_processRef_the_GetProcessDocumentRequestValidator_should_return_a_correct_error_message`

Reply:
I've removed it as redundant